components/courses/app/controllers/courses/students_controller.rb

```diff
@@ -10,7 +10,9 @@ def new
 
     def create
       @student = Student::Create.call(student_params, current_season.id, current_user.id)
-      react_to student.save
+      student_creation_status = student.save
+      NotifySlack::Base.new(student.id).call if student_creation_status
```

Comment:
We should not strict to slack. Tomorrow we will change Slack to SomeOtherNewFashianableChat (Telegram) and we have to rewrite code across all app.

Better solution: create adapter `Notifier`. And it should know what and how to do.